internal/util/K8sUtil.go

```diff
@@ -96,6 +100,37 @@ func (impl K8sUtil) deleteNs(namespace string, client *v12.CoreV1Client) error {
 	return err
 }
 
+func (impl K8sUtil) getargoAppClient(clusterConfig *ClusterConfig) (*rest.RESTClient, error) {
```

Comment:
whats use of this method?

Reply:
was used for create acd apps, not in use now.

Reply:
then please remove it.

Reply:
> then please remove it.

done